### PR TITLE
Increase Ilias Version in the setup

### PR DESCRIPTION
**Pull request by PurHur**
Very important commit.

### CODE REVIEW

**Comment by klees** [comment]
Thanks a bunch!

**Comment by klees** [comment]
Contributor of the year :nerd_face:

File: src/Setup/Condition/PHPExtensionLoadedCondition.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function __construct($which)
             function (Setup\Environment $env) use ($which) : bool {
                 return in_array($which, get_loaded_extensions());
             },
-            "ILIAS 6 requires the PHP extension $which."
+            "ILIAS 7 requires the PHP extension $which."
```
Comment (mjansenDatabay) [inline src/Setup/Condition/PHPExtensionLoadedCondition.php:18]: @klees Is it an option to use `ILIAS_VERSION_NUMERIC` here?

Comment (klees) [reply src/Setup/Condition/PHPExtensionLoadedCondition.php:18]: Hi @mjansenDatabay!

Yes, it is. I just didn't know of that. Out of curiosity: Where is it defined and who bumps it?

Thanks!

Comment (mjansenDatabay) [reply src/Setup/Condition/PHPExtensionLoadedCondition.php:18]: @klees I created a PR here https://github.com/ILIAS-eLearning/ILIAS/pull/3204 , where I answered your questions as well.

**Pull request merged**